ui/pages/login.tsx

```diff
@@ -103,13 +106,25 @@ const Login: NextPage = () => {
         }
         return "password";
       };
+      const method = getMethod();
+
+      const isPasswordMissing = !(values as UpdateLoginFlowWithPasswordMethod)
+        .password;
+
+      const setEmptyPassword = () => {
+        (values as UpdateLoginFlowWithPasswordMethod).password = "";
+      };
```

Comment:
This is in the submit handler, so it will only run once after user interaction. In JS defining a function cost is near zero. Moving it outside the component will reduce cohesion, so I'd suggest keeping it as it is.